data/deep missions.txt

```diff
@@ -351,83 +360,86 @@ mission "Deep: Remnant 0"
 			branch friendly
 				has "event: ember waste label"
-			`	You tell Ivan all that you saw on the other side of the wormhole, and how you found human like beings who attacked you after you ran from them.`
+			`	You tell Ivan all that you saw on the other side of the wormhole, and how you found strange-looking humans who communicate in song and sign language, seemed obsessed with blood, and attacked you after you wouldn't let them have some of yours.`
 				goto end
 			
 			label friendly
-			`	You tell Ivan about how the region of space that the wormhole leads to is known as the Ember Waste, and that it is inhabited by a group of friendly humans who fled human space during the Alpha Wars and have advanced independently of humanity for centuries.`
+			`	You tell Ivan about how the region of space beyond the wormhole is known as the "Ember Waste"; that it is inhabited by a group of friendly humans who fled there during the Alpha Wars and have advanced independently for centuries. You mention their strange culture, sharing what you can remember about the songs the Remnant people sang.`
 			
 			label end
-			`	"This is very interesting, and would turn out to be an amazing discovery if true. Please, bring us the Quantum Keystones so that we might try to understand why they allow entry into the wormhole, and later we may attempt to learn more about these beings that you describe."`
+			`	"This is very interesting, and would turn out to be an amazing discovery if true. Please, bring us the Quantum Keystones so that we might try to understand the properties which allow entry into the wormhole's threshold. Later we will learn more about these inhabitants you describe."`
 			choice
 				`	"I would be glad to help."`
 					accept
 				`	"Sorry, I don't want to help with this."`
 					decline
-
+	
+	on visit
+		dialog `You do not have the 10 Quantum Keystones that Ivan asked you to purchase in Hai space. You should return here after obtaining them.`
+	
 	on complete
 		outfit "Quantum Keystone" -10
 		event "deep: keystone research" 47
 		payment 60000
-		conversation # Maria will be introduced in Deep: TMBR
-			`Upon returning to <planet>, you learn that Ivan has left for his research lab on Valhalla, but Maria, who turns out to also be on the team working on the spacial anomaly, is here to retrieve the keystones. She hands you <payment>, enough to pay for half of the cost of the stones.`
-			`	"Thank you for buying these keystones for us. It may take us a month or two to fully study them. We will make sure to contact you when we are in need of your assistance again."`
+		conversation # Maria will be introduced in Deep: TMBR. She could be postdoc / partner scientist etc.
+			`Upon returning to <planet>, you learn that Ivan has left for his research lab on Valhalla, but his research assistant Maria is waiting to retrieve the keystones. She hands you <payment>, enough to pay for half of the cost of the stones.`
```

Comment:
Deep: TMBR must be done before Deep: Remnant can be started, and Ivan is introduced in this string, so Maria can't be already introduced as being Ivan's assistant. Unless I don't use the Maria character in Deep: TMBR, this doesn't really make sense.